Doc/c-api/typeobj.rst

```diff
@@ -1558,6 +1558,8 @@ and :c:type:`PyType_Type` effectively act as defaults.)
    :c:member:`~PyTypeObject.tp_init` function is called; if :c:member:`~PyTypeObject.tp_new` returns an instance of a
    subtype of the original type, the subtype's :c:member:`~PyTypeObject.tp_init` is called.
 
+   Return ``0`` on success, or return ``-1`` and sets an exception on error.
```

Comment:
Change _return_ to _returns_ to match _sets_ (or change _sets_ → _set_)

Reply:
@vadmium done.